src/backend/base/langflow/services/database/service.py

```diff
@@ -182,14 +183,18 @@ def on_connection(self, dbapi_connection, _connection_record) -> None:
 
     @asynccontextmanager
     async def with_session(self):
-        async with AsyncSession(self.engine, expire_on_commit=False) as session:
-            # Start of Selection
-            try:
-                yield session
-            except exc.SQLAlchemyError as db_exc:
-                logger.error(f"Database error during session scope: {db_exc}")
-                await session.rollback()
-                raise
+        if self.settings_service.settings.use_noop_database:
+            logger.info("Running with NOOP database session. All DB operations are disabled.")
```

Comment:
Probably can move to debug, this may cause spam. Or move to once during initialization.

Reply:
You are right.